src/murmur/DBus.h

```diff
@@ -103,6 +103,12 @@ struct LogEntry {
 Q_DECLARE_METATYPE(LogEntry);
 Q_DECLARE_METATYPE(QList<LogEntry>);
 
+class MurmurDBusConnectionHolder {
```

Comment:
I guess I just wanted to avoid having to deref a pointer qdbc directly (*qdbc) in the QDBusInterface constructors... I'll try.